### ISSUE TITLE
Staging Review Finding: Alt text for Onsite Notification [Assistive tech and device support] Pages require sensory characteristics to be understood or operable. (11.04.1)

### ISSUE THREAD

**Issue by chaseakirby** [department-of-veterans-affairs/va.gov-team#39474]
#### VFS team name
Authenticated Experience

#### VFS product name
My VA Dashboard

#### VFS feature name
Payment Information

#### Point of Contact/Reviewers
Josh Kim (@joshkimux) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Pages require sensory characteristics to be understood or operable.

### Issue Details
No alternative is provided to screen reader users to inform them that content is important (which is demonstrated to sighted users through the "!" icon). If this is not a design system component, I recommend following the advice listed in [ticket 34160](https://github.com/department-of-veterans-affairs/va.gov-team/issues/34160#issuecomment-1061351068).

### VA.gov Experience Standard
[Category Number 11, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.5.5_AAA

---

### Platform Recommendation
Following VA design system guidance, we recommend including a sr-only span to provide an alternative to this icon font. For example: <span class="sr-only">Alert: [or whatever equivalent alternative needed e.g. "Important"]</span>.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by hdjustice** [comment]
4/6/22 - We already have a ticket for adding Alt Text to alert icon for Onsite Notification

**Comment by ajakabcin** [comment]
Per discussion in this [ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/979#issuecomment-1169008253) we it was determined we do not need to address this issue.
